WORKSPACE

```diff
@@ -169,32 +169,6 @@ dev_env_tool(
     win_tool = "msys2-20180531",
 )
 
-# c2hs
-nixpkgs_package(
-    name = "c2hs",
-    attribute_path = "ghcWithC2hs",
-    build_file_content = '''
-
-package(default_visibility = [ "//visibility:public" ])
-
-filegroup(
-    name = "bin",
-    srcs = ["bin/c2hs"],
-)
-exports_files(["bin/ghc-pkg"])
```

Comment:
This is going to break since we use `ghc-pkg` iirc. The proper solution would probably be to get it from the toolchain.

Reply:
This is now using the ghc-pkg from the GHC workspace.